Svelte module resolutions not working.

### Summary

<!-- Please insert a one line summary of the issue below -->

Svelte module resolutions not working. It cannot resolves $lib modules.

Steps to trigger the problem:
1. run `npx sv create my-app` minimal
2. Activate typescript
3. Go to +page.svelte
4. Try to import module from '$lib'

Actual Behavior:

![Image](https://github.com/user-attachments/assets/3a3672f9-dd86-4b0a-96c0-62d596138c30)

Expected Behavior:

Two days ago (before I updated to 0.175.5) it works fine, the $lib module resolution works as expected.

  Did you check your Zed.log to see if there is any relevant details there?
    - Log:

  <details>
  <summary>See inside for LOG</summary>

  ```shell
2025-02-27T18:27:39.582410574+07:00 [INFO] Restarting process, using app path: "~/.local/zed.app/libexec/zed-editor"
2025-02-27T18:27:39.598459273+07:00 [ERROR] oneshot canceled
2025-02-27T18:27:39.600522908+07:00 [ERROR] oneshot canceled
2025-02-27T18:27:39.600613119+07:00 [ERROR] oneshot canceled
2025-02-27T18:27:39.600681598+07:00 [ERROR] oneshot canceled
2025-02-27T18:27:39.60672248+07:00 [ERROR] oneshot canceled
2025-02-27T18:27:39.60679966+07:00 [ERROR] oneshot canceled
2025-02-27T18:27:39.700532197+07:00 [INFO] ========== starting zed ==========
2025-02-27T18:27:39.751086504+07:00 [INFO] Presentation support: "VK_KHR_wayland_surface"
2025-02-27T18:27:39.751213716+07:00 [INFO] Presentation support: "VK_KHR_xcb_surface"
2025-02-27T18:27:39.75129272+07:00 [INFO] Presentation support: "VK_KHR_xlib_surface"
2025-02-27T18:27:39.751376158+07:00 [INFO] Enabling Vulkan Portability
2025-02-27T18:27:39.751438325+07:00 [INFO] Enabling color space support
2025-02-27T18:27:39.763738726+07:00 [INFO] Adapter: "Intel(R) Graphics (RPL-P)"
2025-02-27T18:27:39.763807299+07:00 [INFO] No ray tracing extensions are supported
2025-02-27T18:27:39.765740886+07:00 [INFO] Opening main db
2025-02-27T18:27:39.765776458+07:00 [INFO] perform;
2025-02-27T18:27:39.765826592+07:00 [INFO] authenticate;
2025-02-27T18:27:39.7658789+07:00 [INFO] read_command;
2025-02-27T18:27:39.766035745+07:00 [INFO] send_secondary_commands;
2025-02-27T18:27:39.766474421+07:00 [INFO] new;
2025-02-27T18:27:39.766571434+07:00 [INFO] socket reader;
2025-02-27T18:27:39.766608746+07:00 [INFO] read_socket;
2025-02-27T18:27:39.766663229+07:00 [INFO] read_socket;
2025-02-27T18:27:39.766712261+07:00 [INFO] Opening main db
2025-02-27T18:27:39.766921473+07:00 [INFO] read_socket;
2025-02-27T18:27:39.767078443+07:00 [INFO] read_socket;
2025-02-27T18:27:39.767172626+07:00 [INFO] read_socket;
2025-02-27T18:27:39.767412349+07:00 [INFO] read_socket;
2025-02-27T18:27:39.767458438+07:00 [INFO] keep_updated;
2025-02-27T18:27:39.767889189+07:00 [INFO] Using git binary path: None
2025-02-27T18:27:39.768419024+07:00 [INFO] read_socket;
2025-02-27T18:27:39.768882055+07:00 [INFO] read_socket;
2025-02-27T18:27:39.769389817+07:00 [INFO] read_socket;
2025-02-27T18:27:39.769485061+07:00 [INFO] read_socket;
2025-02-27T18:27:39.769594814+07:00 [INFO] read_socket;
2025-02-27T18:27:39.769677319+07:00 [INFO] read_socket;
2025-02-27T18:27:39.769734873+07:00 [INFO] read_socket;
2025-02-27T18:27:39.769804689+07:00 [INFO] read_socket;
2025-02-27T18:27:39.769873674+07:00 [INFO] read_socket;
2025-02-27T18:27:39.783146855+07:00 [INFO] No prompt template overrides directory found at ~/.local/share/zed/prompt_overrides. Using built-in prompts.
2025-02-27T18:27:39.783546916+07:00 [INFO] extensions updated. loading 11, reloading 0, unloading 0
2025-02-27T18:27:39.785036298+07:00 [INFO] activate is not implemented on Linux, ignoring the call
2025-02-27T18:27:39.786263907+07:00 [INFO] Opening main db
2025-02-27T18:27:39.78647705+07:00 [INFO] perform;
2025-02-27T18:27:39.786500232+07:00 [INFO] authenticate;
2025-02-27T18:27:39.786531356+07:00 [INFO] read_command;
2025-02-27T18:27:39.786574455+07:00 [INFO] send_secondary_commands;
2025-02-27T18:27:39.787028631+07:00 [INFO] socket reader;
2025-02-27T18:27:39.787050281+07:00 [INFO] read_socket;
2025-02-27T18:27:39.787095264+07:00 [INFO] read_socket;
2025-02-27T18:27:39.790681667+07:00 [INFO] read_socket;
2025-02-27T18:27:39.791753626+07:00 [INFO] read_socket;
2025-02-27T18:27:39.791922868+07:00 [INFO] read_socket;
2025-02-27T18:27:39.792181237+07:00 [INFO] read_socket;
2025-02-27T18:27:39.792205596+07:00 [INFO] new;
2025-02-27T18:27:39.792328674+07:00 [INFO] read_socket;
2025-02-27T18:27:39.792414704+07:00 [INFO] read_socket;
2025-02-27T18:27:39.792494447+07:00 [INFO] read_socket;
2025-02-27T18:27:39.792749129+07:00 [INFO] read_socket;
2025-02-27T18:27:39.792808081+07:00 [INFO] keep_updated;
2025-02-27T18:27:39.792978609+07:00 [INFO] read_socket;
2025-02-27T18:27:39.793071155+07:00 [INFO] read_socket;
2025-02-27T18:27:39.793142873+07:00 [INFO] read_socket;
2025-02-27T18:27:39.794890258+07:00 [INFO] set status on client 0: Authenticating
2025-02-27T18:27:39.795023167+07:00 [INFO] perform;
2025-02-27T18:27:39.795047875+07:00 [INFO] authenticate;
2025-02-27T18:27:39.795079238+07:00 [INFO] read_command;
2025-02-27T18:27:39.795220646+07:00 [INFO] send_secondary_commands;
2025-02-27T18:27:39.795806304+07:00 [INFO] socket reader;
2025-02-27T18:27:39.795826174+07:00 [INFO] read_socket;
2025-02-27T18:27:39.795856164+07:00 [INFO] read_socket;
2025-02-27T18:27:39.798158027+07:00 [INFO] Using surface present mode MAILBOX
2025-02-27T18:27:39.799137112+07:00 [INFO] Creating a descriptor pool for at most 16 sets
2025-02-27T18:27:39.7993031+07:00 [INFO] Creating a descriptor pool for at most 16 sets
2025-02-27T18:27:39.799464596+07:00 [INFO] Initializing Blade pipelines for surface SurfaceInfo { format: Bgra8UnormSrgb, alpha: PreMultiplied }
2025-02-27T18:27:39.799605302+07:00 [INFO] read_socket;
2025-02-27T18:27:39.801190397+07:00 [INFO] read_socket;
2025-02-27T18:27:39.801357058+07:00 [INFO] read_socket;
2025-02-27T18:27:39.801597839+07:00 [INFO] read_socket;
2025-02-27T18:27:39.801613372+07:00 [INFO] new;
2025-02-27T18:27:39.801733785+07:00 [INFO] read_socket;
2025-02-27T18:27:39.801797311+07:00 [INFO] read_socket;
2025-02-27T18:27:39.801860079+07:00 [INFO] read_socket;
2025-02-27T18:27:39.802064889+07:00 [INFO] read_socket;
2025-02-27T18:27:39.802105199+07:00 [INFO] keep_updated;
2025-02-27T18:27:39.802291764+07:00 [INFO] read_socket;
2025-02-27T18:27:39.802413314+07:00 [INFO] read_socket;
2025-02-27T18:27:39.802468089+07:00 [INFO] read_socket;
2025-02-27T18:27:39.816705864+07:00 [INFO] Using GPU: GpuSpecs { is_software_emulated: false, device_name: "Intel(R) Graphics (RPL-P)", driver_name: "Intel open-source Mesa driver", driver_info: "Mesa 24.2.8-1ubuntu1~24.04.1" }
2025-02-27T18:27:39.817316503+07:00 [INFO] Prettier config file ".prettierignore" changed, reloading prettier instances for worktree 4294967342
2025-02-27T18:27:39.817375613+07:00 [INFO] set status on client 300014: Connecting
2025-02-27T18:27:39.81788876+07:00 [INFO] Opening main db
2025-02-27T18:27:39.825858121+07:00 [INFO] set environment variables from shell:/bin/bash, path:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.cargo/bin:~/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin
2025-02-27T18:27:39.833267838+07:00 [INFO] Using surface present mode MAILBOX
2025-02-27T18:27:39.833302542+07:00 [INFO] Retaining current format: Bgra8UnormSrgb
2025-02-27T18:27:39.837119973+07:00 [INFO] open paths ["~/.config/zed/settings.json"]
2025-02-27T18:27:39.837831477+07:00 [INFO] Opening main db
2025-02-27T18:27:39.944436332+07:00 [INFO] attempting to start language server "json-language-server", path: "~/.config/zed/settings.json", id: 1
2025-02-27T18:27:39.944478088+07:00 [INFO] Initializing default prettier with plugins {}
2025-02-27T18:27:39.983890025+07:00 [INFO] Initializing default prettier with plugins {}
2025-02-27T18:27:39.985399033+07:00 [INFO] attempting to start language server "tailwindcss-language-server", path: "~/code/username/optilab/svelte-qwik/sveltekit-app", id: 2
2025-02-27T18:27:39.985462322+07:00 [INFO] attempting to start language server "eslint", path: "~/code/username/optilab/svelte-qwik/sveltekit-app", id: 3
2025-02-27T18:27:39.985507814+07:00 [INFO] Initializing default prettier with plugins {"prettier-plugin-svelte"}
2025-02-27T18:27:39.986076755+07:00 [INFO] fetching latest version of language server "eslint"
2025-02-27T18:27:39.986114104+07:00 [INFO] downloading language server "eslint"
2025-02-27T18:27:39.986328655+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:40.015467567+07:00 [INFO] using project environment variables shell launched in "~/.config/zed/settings.json". PATH="~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.cargo/bin:~/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin"
2025-02-27T18:27:40.0156121+07:00 [INFO] fetching latest version of language server "json-language-server"
2025-02-27T18:27:40.03420936+07:00 [INFO] using project environment variables shell launched in "~/code/username/optilab/svelte-qwik/sveltekit-app". PATH="~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.cargo/bin:~/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin"
2025-02-27T18:27:40.034611077+07:00 [INFO] fetching latest version of language server "tailwindcss-language-server"
2025-02-27T18:27:40.061806628+07:00 [INFO] Initializing default prettier with plugins {"prettier-plugin-svelte"}
2025-02-27T18:27:40.061857844+07:00 [INFO] Initializing default prettier with plugins {"prettier-plugin-svelte"}
2025-02-27T18:27:40.069864074+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:40.070333733+07:00 [INFO] starting language server process. binary path: "~/.local/share/zed/node/node-v22.5.1-linux-x64/bin/node", working directory: "~/code/username/optilab/svelte-qwik/sveltekit-app", args: ["--max-old-space-size=8192", "~/.local/share/zed/languages/eslint/vscode-eslint-2.4.4/vscode-eslint/server/out/eslintServer.js", "--stdio"]
2025-02-27T18:27:40.119810182+07:00 [INFO] Initializing default prettier with plugins {"prettier-plugin-svelte"}
2025-02-27T18:27:40.119840961+07:00 [INFO] Initializing default prettier with plugins {"prettier-plugin-svelte"}
2025-02-27T18:27:40.139793419+07:00 [WARN] unhandled capability registration: Registration { id: "a9a2ab76-caa8-4797-836b-1c8f6056f70c", method: "workspace/didChangeWorkspaceFolders", register_options: Some(Object {}) }
2025-02-27T18:27:40.144565678+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:40.216612357+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:40.289574323+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:40.457602505+07:00 [INFO] connected to rpc endpoint https://collab.zed.dev/rpc
2025-02-27T18:27:40.611471039+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:40.685329484+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:40.761224159+07:00 [INFO] language server "json-language-server" is already installed
2025-02-27T18:27:40.76130319+07:00 [INFO] starting language server process. binary path: "~/.local/share/zed/node/node-v22.5.1-linux-x64/bin/node", working directory: "~/.config/zed", args: ["~/.local/share/zed/languages/json-language-server/node_modules/vscode-langservers-extracted/bin/vscode-json-language-server", "--stdio"]
2025-02-27T18:27:40.813677137+07:00 [INFO] Installing default prettier and plugins: [("prettier-plugin-svelte", "3.3.3"), ("prettier", "3.5.2")]
2025-02-27T18:27:40.81385627+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:40.892728307+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:40.935742334+07:00 [INFO] Language server with id 3 sent unhandled notification eslint/status:
{
"uri": "file://~/code/username/optilab/svelte-qwik/sveltekit-app/src/routes/+page.svelte",
"state": 3
}
2025-02-27T18:27:40.964023031+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:41.008143798+07:00 [INFO] attempting to start language server "svelte-language-server", path: "~/code/username/optilab/svelte-qwik/sveltekit-app", id: 4
2025-02-27T18:27:41.008212126+07:00 [INFO] Initializing default prettier with plugins {"prettier-plugin-svelte"}
2025-02-27T18:27:41.008245443+07:00 [INFO] Initializing default prettier with plugins {"prettier-plugin-svelte"}
2025-02-27T18:27:41.036250699+07:00 [INFO] language server "tailwindcss-language-server" is already installed
2025-02-27T18:27:41.036325563+07:00 [INFO] starting language server process. binary path: "~/.local/share/zed/node/node-v22.5.1-linux-x64/bin/node", working directory: "~/code/username/optilab/svelte-qwik/sveltekit-app", args: ["~/.local/share/zed/languages/tailwindcss-language-server/node_modules/.bin/tailwindcss-language-server", "--stdio"]
2025-02-27T18:27:41.03645647+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:41.056392859+07:00 [INFO] Language server with id 3 sent unhandled notification eslint/status:
{
"uri": "file://~/code/username/optilab/svelte-qwik/sveltekit-app/src/routes/+page.svelte",
"state": 3
}
2025-02-27T18:27:41.516381535+07:00 [INFO] add_connection;
2025-02-27T18:27:41.516606914+07:00 [INFO] set status on client 300014: Connected { peer_id: PeerId { owner_id: 683, id: 1761870 }, connection_id: ConnectionId { owner_id: 0, id: 0 } }
2025-02-27T18:27:41.733689878+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:41.804585616+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:41.877773654+07:00 [INFO] perform;
2025-02-27T18:27:41.877842733+07:00 [INFO] authenticate;
2025-02-27T18:27:41.877886252+07:00 [INFO] perform;
2025-02-27T18:27:41.877934092+07:00 [INFO] authenticate;
2025-02-27T18:27:41.877965997+07:00 [INFO] read_command;
2025-02-27T18:27:41.877998483+07:00 [INFO] send_secondary_commands;
2025-02-27T18:27:41.878019572+07:00 [INFO] perform;
2025-02-27T18:27:41.878048982+07:00 [INFO] authenticate;
2025-02-27T18:27:41.878076937+07:00 [INFO] read_command;
2025-02-27T18:27:41.878103095+07:00 [INFO] send_secondary_commands;
2025-02-27T18:27:41.878126776+07:00 [INFO] perform;
2025-02-27T18:27:41.878174234+07:00 [INFO] authenticate;
2025-02-27T18:27:41.878225352+07:00 [INFO] read_command;
2025-02-27T18:27:41.878329134+07:00 [INFO] perform;
2025-02-27T18:27:41.878374587+07:00 [INFO] authenticate;
2025-02-27T18:27:41.878412131+07:00 [INFO] read_command;
2025-02-27T18:27:41.878451604+07:00 [INFO] read_command;
2025-02-27T18:27:41.878486561+07:00 [INFO] send_secondary_commands;
2025-02-27T18:27:41.878506519+07:00 [INFO] socket reader;
2025-02-27T18:27:41.878522184+07:00 [INFO] read_socket;
2025-02-27T18:27:41.878554528+07:00 [INFO] send_secondary_commands;
2025-02-27T18:27:41.878574828+07:00 [INFO] read_socket;
2025-02-27T18:27:41.878593172+07:00 [INFO] send_secondary_commands;
2025-02-27T18:27:41.878903131+07:00 [INFO] socket reader;
2025-02-27T18:27:41.878923896+07:00 [INFO] read_socket;
2025-02-27T18:27:41.878960276+07:00 [INFO] read_socket;
2025-02-27T18:27:41.879099518+07:00 [INFO] socket reader;
2025-02-27T18:27:41.879125179+07:00 [INFO] read_socket;
2025-02-27T18:27:41.87917359+07:00 [INFO] read_socket;
2025-02-27T18:27:41.879305997+07:00 [INFO] socket reader;
2025-02-27T18:27:41.879329008+07:00 [INFO] read_socket;
2025-02-27T18:27:41.879382706+07:00 [INFO] read_socket;
2025-02-27T18:27:41.879608449+07:00 [INFO] socket reader;
2025-02-27T18:27:41.879644666+07:00 [INFO] read_socket;
2025-02-27T18:27:41.879666821+07:00 [INFO] starting language server process. binary path: "~/.local/share/zed/node/node-v22.5.1-linux-x64/bin/node", working directory: "~/code/username/optilab/svelte-qwik/sveltekit-app", args: ["~/.local/share/zed/extensions/work/svelte/node_modules/svelte-language-server/bin/server.js", "--stdio"]
2025-02-27T18:27:41.879711737+07:00 [INFO] read_socket;
2025-02-27T18:27:41.882279307+07:00 [INFO] read_socket;
2025-02-27T18:27:41.884195681+07:00 [INFO] Initializing default prettier with plugins {"prettier-plugin-svelte"}
2025-02-27T18:27:41.884230799+07:00 [INFO] Initializing default prettier with plugins {"prettier-plugin-svelte"}
2025-02-27T18:27:41.8853276+07:00 [INFO] read_socket;
2025-02-27T18:27:41.889121102+07:00 [INFO] read_socket;
2025-02-27T18:27:41.891942634+07:00 [INFO] read_socket;
2025-02-27T18:27:41.894017927+07:00 [INFO] read_socket;
2025-02-27T18:27:41.894093539+07:00 [INFO] read_socket;
2025-02-27T18:27:41.894116643+07:00 [INFO] read_socket;
2025-02-27T18:27:41.894139598+07:00 [INFO] read_socket;
2025-02-27T18:27:41.894165605+07:00 [INFO] read_socket;
2025-02-27T18:27:41.894386879+07:00 [INFO] read_socket;
2025-02-27T18:27:41.894407714+07:00 [INFO] read_socket;
2025-02-27T18:27:41.894598614+07:00 [INFO] read_socket;
2025-02-27T18:27:41.894716051+07:00 [INFO] read_socket;
2025-02-27T18:27:41.89482275+07:00 [INFO] read_socket;
2025-02-27T18:27:41.894911095+07:00 [INFO] read_socket;
2025-02-27T18:27:41.895141324+07:00 [INFO] read_socket;
2025-02-27T18:27:41.895169275+07:00 [INFO] read_socket;
2025-02-27T18:27:41.89519189+07:00 [INFO] read_socket;
2025-02-27T18:27:41.895643814+07:00 [INFO] read_socket;
2025-02-27T18:27:41.896004717+07:00 [INFO] read_socket;
2025-02-27T18:27:41.897446989+07:00 [INFO] Initialized prettier with plugins: {"prettier-plugin-svelte"}
2025-02-27T18:27:41.900174086+07:00 [INFO] Language server with id 2 sent unhandled notification @/tailwindCSS/projectInitialized:
[
null
]
2025-02-27T18:27:41.91650429+07:00 [INFO] Language server with id 2 sent unhandled notification @/tailwindCSS/clearColors:
[
null
]
2025-02-27T18:27:41.935908441+07:00 [INFO] Language server with id 3 sent unhandled notification eslint/status:
{
"uri": "file://~/code/username/optilab/svelte-qwik/sveltekit-app/src/routes/+page.svelte",
"state": 3
}
2025-02-27T18:27:48.949888195+07:00 [INFO] Starting prettier at path "~/.local/share/zed/prettier"
2025-02-27T18:27:48.950471454+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:49.038856324+07:00 [INFO] starting language server process. binary path: "~/.local/share/zed/node/node-v22.5.1-linux-x64/bin/node", working directory: "~/.local/share/zed/prettier", args: ["~/.local/share/zed/prettier/prettier_server.js", "~/.local/share/zed/prettier"]
2025-02-27T18:27:49.105710511+07:00 [INFO] Started default prettier in "~/.local/share/zed/prettier"
2025-02-27T18:27:49.264298+07:00 [INFO] Language server with id 2 sent unhandled notification @/tailwindCSS/clearColors:
[
null
]
2025-02-27T18:27:49.30189167+07:00 [INFO] Language server with id 3 sent unhandled notification eslint/status:
{
"uri": "file://~/code/username/optilab/svelte-qwik/sveltekit-app/src/routes/+page.svelte",
"state": 3
}
2025-02-27T18:27:53.553824252+07:00 [INFO] using project environment variables shell launched in "~/.local/share/zed/extensions/work/svelte/node_modules/typescript/lib/lib.dom.d.ts". PATH="~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.cargo/bin:~/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin"
2025-02-27T18:27:53.60386348+07:00 [INFO] Language server with id 2 sent unhandled notification @/tailwindCSS/clearColors:
[
null
]
2025-02-27T18:27:53.639265623+07:00 [INFO] Language server with id 3 sent unhandled notification eslint/status:
{
"uri": "file://~/code/username/optilab/svelte-qwik/sveltekit-app/src/routes/+page.svelte",
"state": 3
}
2025-02-27T18:27:56.888065923+07:00 [INFO] Restarting process, using app path: "~/.local/zed.app/libexec/zed-editor"
2025-02-27T18:27:56.906105533+07:00 [ERROR] oneshot canceled
2025-02-27T18:27:56.906226846+07:00 [ERROR] oneshot canceled
2025-02-27T18:27:56.908432179+07:00 [ERROR] oneshot canceled
2025-02-27T18:27:56.908488265+07:00 [ERROR] oneshot canceled
2025-02-27T18:27:56.914193002+07:00 [ERROR] oneshot canceled
2025-02-27T18:27:57.000736315+07:00 [INFO] ========== starting zed ==========
2025-02-27T18:27:57.042016641+07:00 [INFO] Presentation support: "VK_KHR_wayland_surface"
2025-02-27T18:27:57.04206803+07:00 [INFO] Presentation support: "VK_KHR_xcb_surface"
2025-02-27T18:27:57.042098026+07:00 [INFO] Presentation support: "VK_KHR_xlib_surface"
2025-02-27T18:27:57.042131192+07:00 [INFO] Enabling Vulkan Portability
2025-02-27T18:27:57.0421557+07:00 [INFO] Enabling color space support
2025-02-27T18:27:57.048999321+07:00 [INFO] Adapter: "Intel(R) Graphics (RPL-P)"
2025-02-27T18:27:57.049073129+07:00 [INFO] No ray tracing extensions are supported
2025-02-27T18:27:57.051161408+07:00 [INFO] perform;
2025-02-27T18:27:57.051217553+07:00 [INFO] authenticate;
2025-02-27T18:27:57.051286649+07:00 [INFO] read_command;
2025-02-27T18:27:57.051380119+07:00 [INFO] Opening main db
2025-02-27T18:27:57.051445012+07:00 [INFO] send_secondary_commands;
2025-02-27T18:27:57.05202948+07:00 [INFO] new;
2025-02-27T18:27:57.052075844+07:00 [INFO] socket reader;
2025-02-27T18:27:57.052120969+07:00 [INFO] read_socket;
2025-02-27T18:27:57.052194257+07:00 [INFO] read_socket;
2025-02-27T18:27:57.052368295+07:00 [INFO] read_socket;
2025-02-27T18:27:57.052520118+07:00 [INFO] Opening main db
2025-02-27T18:27:57.052573197+07:00 [INFO] read_socket;
2025-02-27T18:27:57.05268708+07:00 [INFO] read_socket;
2025-02-27T18:27:57.052954043+07:00 [INFO] read_socket;
2025-02-27T18:27:57.053004497+07:00 [INFO] keep_updated;
2025-02-27T18:27:57.054069835+07:00 [INFO] read_socket;
2025-02-27T18:27:57.054105925+07:00 [INFO] Using git binary path: None
2025-02-27T18:27:57.054947299+07:00 [INFO] read_socket;
2025-02-27T18:27:57.055753209+07:00 [INFO] read_socket;
2025-02-27T18:27:57.055909285+07:00 [INFO] read_socket;
2025-02-27T18:27:57.056065571+07:00 [INFO] read_socket;
2025-02-27T18:27:57.05621557+07:00 [INFO] read_socket;
2025-02-27T18:27:57.05633321+07:00 [INFO] read_socket;
2025-02-27T18:27:57.056487754+07:00 [INFO] read_socket;
2025-02-27T18:27:57.056622739+07:00 [INFO] read_socket;
2025-02-27T18:27:57.074597+07:00 [INFO] No prompt template overrides directory found at ~/.local/share/zed/prompt_overrides. Using built-in prompts.
2025-02-27T18:27:57.075037334+07:00 [INFO] extensions updated. loading 11, reloading 0, unloading 0
2025-02-27T18:27:57.07633673+07:00 [INFO] activate is not implemented on Linux, ignoring the call
2025-02-27T18:27:57.077513696+07:00 [INFO] Opening main db
2025-02-27T18:27:57.077727058+07:00 [INFO] perform;
2025-02-27T18:27:57.077746064+07:00 [INFO] authenticate;
2025-02-27T18:27:57.077767168+07:00 [INFO] read_command;
2025-02-27T18:27:57.077907768+07:00 [INFO] send_secondary_commands;
2025-02-27T18:27:57.078451279+07:00 [INFO] socket reader;
2025-02-27T18:27:57.078473898+07:00 [INFO] read_socket;
2025-02-27T18:27:57.078519334+07:00 [INFO] read_socket;
2025-02-27T18:27:57.082424474+07:00 [INFO] read_socket;
2025-02-27T18:27:57.083607977+07:00 [INFO] read_socket;
2025-02-27T18:27:57.083899927+07:00 [INFO] read_socket;
2025-02-27T18:27:57.084172931+07:00 [INFO] read_socket;
2025-02-27T18:27:57.084216641+07:00 [INFO] new;
2025-02-27T18:27:57.08437861+07:00 [INFO] read_socket;
2025-02-27T18:27:57.08448646+07:00 [INFO] read_socket;
2025-02-27T18:27:57.084600104+07:00 [INFO] read_socket;
2025-02-27T18:27:57.084922412+07:00 [INFO] read_socket;
2025-02-27T18:27:57.08498792+07:00 [INFO] keep_updated;
2025-02-27T18:27:57.085343239+07:00 [INFO] read_socket;
2025-02-27T18:27:57.085494453+07:00 [INFO] read_socket;
2025-02-27T18:27:57.085592415+07:00 [INFO] read_socket;
2025-02-27T18:27:57.08645246+07:00 [INFO] set status on client 0: Authenticating
2025-02-27T18:27:57.086621837+07:00 [INFO] perform;
2025-02-27T18:27:57.086641105+07:00 [INFO] authenticate;
2025-02-27T18:27:57.086663025+07:00 [INFO] read_command;
2025-02-27T18:27:57.086732315+07:00 [INFO] send_secondary_commands;
2025-02-27T18:27:57.087188782+07:00 [INFO] socket reader;
2025-02-27T18:27:57.087210704+07:00 [INFO] read_socket;
2025-02-27T18:27:57.087250723+07:00 [INFO] read_socket;
2025-02-27T18:27:57.090134217+07:00 [INFO] Using surface present mode MAILBOX
2025-02-27T18:27:57.090590254+07:00 [INFO] read_socket;
2025-02-27T18:27:57.091108742+07:00 [INFO] Creating a descriptor pool for at most 16 sets
2025-02-27T18:27:57.091264154+07:00 [INFO] Creating a descriptor pool for at most 16 sets
2025-02-27T18:27:57.091417598+07:00 [INFO] Initializing Blade pipelines for surface SurfaceInfo { format: Bgra8UnormSrgb, alpha: PreMultiplied }
2025-02-27T18:27:57.09211375+07:00 [INFO] read_socket;
2025-02-27T18:27:57.0924559+07:00 [INFO] read_socket;
2025-02-27T18:27:57.092848155+07:00 [INFO] read_socket;
2025-02-27T18:27:57.092891111+07:00 [INFO] new;
2025-02-27T18:27:57.09309127+07:00 [INFO] read_socket;
2025-02-27T18:27:57.093199736+07:00 [INFO] read_socket;
2025-02-27T18:27:57.093291325+07:00 [INFO] read_socket;
2025-02-27T18:27:57.093577399+07:00 [INFO] read_socket;
2025-02-27T18:27:57.093638685+07:00 [INFO] keep_updated;
2025-02-27T18:27:57.093838438+07:00 [INFO] read_socket;
2025-02-27T18:27:57.093941363+07:00 [INFO] read_socket;
2025-02-27T18:27:57.094027328+07:00 [INFO] read_socket;
2025-02-27T18:27:57.101487592+07:00 [INFO] Using GPU: GpuSpecs { is_software_emulated: false, device_name: "Intel(R) Graphics (RPL-P)", driver_name: "Intel open-source Mesa driver", driver_info: "Mesa 24.2.8-1ubuntu1~24.04.1" }
2025-02-27T18:27:57.101872972+07:00 [INFO] set status on client 300014: Connecting
2025-02-27T18:27:57.102342577+07:00 [INFO] Opening main db
2025-02-27T18:27:57.110453391+07:00 [INFO] open paths ["~/.config/zed/settings.json"]
2025-02-27T18:27:57.118135806+07:00 [INFO] set environment variables from shell:/bin/bash, path:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.cargo/bin:~/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin
2025-02-27T18:27:57.125111194+07:00 [INFO] Using surface present mode MAILBOX
2025-02-27T18:27:57.125140058+07:00 [INFO] Retaining current format: Bgra8UnormSrgb
2025-02-27T18:27:57.126193669+07:00 [INFO] Prettier config file ".prettierignore" changed, reloading prettier instances for worktree 4294967342
2025-02-27T18:27:57.126465865+07:00 [INFO] Opening main db
2025-02-27T18:27:57.23705653+07:00 [INFO] attempting to start language server "json-language-server", path: "~/.config/zed/settings.json", id: 1
2025-02-27T18:27:57.237096439+07:00 [INFO] Initializing default prettier with plugins {}
2025-02-27T18:27:57.272855054+07:00 [INFO] Initializing default prettier with plugins {}
2025-02-27T18:27:57.274819516+07:00 [INFO] attempting to start language server "tailwindcss-language-server", path: "~/code/username/optilab/svelte-qwik/sveltekit-app", id: 2
2025-02-27T18:27:57.274877405+07:00 [INFO] attempting to start language server "eslint", path: "~/code/username/optilab/svelte-qwik/sveltekit-app", id: 3
2025-02-27T18:27:57.274922462+07:00 [INFO] Initializing default prettier with plugins {"prettier-plugin-svelte"}
2025-02-27T18:27:57.275582661+07:00 [INFO] fetching latest version of language server "eslint"
2025-02-27T18:27:57.275639581+07:00 [INFO] downloading language server "eslint"
2025-02-27T18:27:57.275968761+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:57.289731595+07:00 [INFO] using project environment variables shell launched in "~/.config/zed/settings.json". PATH="~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.cargo/bin:~/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin"
2025-02-27T18:27:57.289847867+07:00 [INFO] fetching latest version of language server "json-language-server"
2025-02-27T18:27:57.338555037+07:00 [INFO] Initializing default prettier with plugins {"prettier-plugin-svelte"}
2025-02-27T18:27:57.338602194+07:00 [INFO] Initializing default prettier with plugins {"prettier-plugin-svelte"}
2025-02-27T18:27:57.340678898+07:00 [ERROR] persisting editor selections for editor 12884902048, workspace WorkspaceId(17)

Caused by:
  Sqlite call failed with code 787 and message: Some("FOREIGN KEY constraint failed")
2025-02-27T18:27:57.343847015+07:00 [INFO] using project environment variables shell launched in "~/code/username/optilab/svelte-qwik/sveltekit-app". PATH="~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.local/bin:~/.proto/shims:~/.proto/bin:~/.local/bin:~/.cargo/bin:~/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin"
2025-02-27T18:27:57.344121434+07:00 [INFO] fetching latest version of language server "tailwindcss-language-server"
2025-02-27T18:27:57.361497643+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:57.361885704+07:00 [INFO] starting language server process. binary path: "~/.local/share/zed/node/node-v22.5.1-linux-x64/bin/node", working directory: "~/code/username/optilab/svelte-qwik/sveltekit-app", args: ["--max-old-space-size=8192", "~/.local/share/zed/languages/eslint/vscode-eslint-2.4.4/vscode-eslint/server/out/eslintServer.js", "--stdio"]
2025-02-27T18:27:57.408811395+07:00 [INFO] Initializing default prettier with plugins {"prettier-plugin-svelte"}
2025-02-27T18:27:57.408842326+07:00 [INFO] Initializing default prettier with plugins {"prettier-plugin-svelte"}
2025-02-27T18:27:57.431585932+07:00 [WARN] unhandled capability registration: Registration { id: "f649294a-9e1b-4323-b99b-d67f95e459d6", method: "workspace/didChangeWorkspaceFolders", register_options: Some(Object {}) }
2025-02-27T18:27:57.432914467+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:57.500698492+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:57.509602479+07:00 [INFO] connected to rpc endpoint https://collab.zed.dev/rpc
2025-02-27T18:27:57.57745105+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:57.768548591+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:57.842381271+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:57.914697285+07:00 [INFO] language server "json-language-server" is already installed
2025-02-27T18:27:57.91479138+07:00 [INFO] starting language server process. binary path: "~/.local/share/zed/node/node-v22.5.1-linux-x64/bin/node", working directory: "~/.config/zed", args: ["~/.local/share/zed/languages/json-language-server/node_modules/vscode-langservers-extracted/bin/vscode-json-language-server", "--stdio"]
2025-02-27T18:27:57.973673444+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:58.043429918+07:00 [INFO] Installing default prettier and plugins: [("prettier-plugin-svelte", "3.3.3"), ("prettier", "3.5.2")]
2025-02-27T18:27:58.046292862+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:58.127426349+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:58.203961443+07:00 [INFO] language server "tailwindcss-language-server" is already installed
2025-02-27T18:27:58.20403416+07:00 [INFO] starting language server process. binary path: "~/.local/share/zed/node/node-v22.5.1-linux-x64/bin/node", working directory: "~/code/username/optilab/svelte-qwik/sveltekit-app", args: ["~/.local/share/zed/languages/tailwindcss-language-server/node_modules/.bin/tailwindcss-language-server", "--stdio"]
2025-02-27T18:27:58.231081543+07:00 [INFO] Language server with id 3 sent unhandled notification eslint/status:
{
"uri": "file://~/code/username/optilab/svelte-qwik/sveltekit-app/src/routes/+page.svelte",
"state": 3
}
2025-02-27T18:27:58.274505585+07:00 [INFO] attempting to start language server "svelte-language-server", path: "~/code/username/optilab/svelte-qwik/sveltekit-app", id: 4
2025-02-27T18:27:58.274598524+07:00 [INFO] Initializing default prettier with plugins {"prettier-plugin-svelte"}
2025-02-27T18:27:58.274625963+07:00 [INFO] Initializing default prettier with plugins {"prettier-plugin-svelte"}
2025-02-27T18:27:58.2757761+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:58.325811422+07:00 [INFO] Language server with id 3 sent unhandled notification eslint/status:
{
"uri": "file://~/code/username/optilab/svelte-qwik/sveltekit-app/src/routes/+page.svelte",
"state": 3
}
2025-02-27T18:27:58.434701502+07:00 [INFO] add_connection;
2025-02-27T18:27:58.43533375+07:00 [INFO] set status on client 300014: Connected { peer_id: PeerId { owner_id: 683, id: 1762126 }, connection_id: ConnectionId { owner_id: 0, id: 0 } }
2025-02-27T18:27:58.781530616+07:00 [INFO] perform;
2025-02-27T18:27:58.781600804+07:00 [INFO] authenticate;
2025-02-27T18:27:58.781642393+07:00 [INFO] perform;
2025-02-27T18:27:58.781672822+07:00 [INFO] authenticate;
2025-02-27T18:27:58.781708506+07:00 [INFO] read_command;
2025-02-27T18:27:58.781744785+07:00 [INFO] send_secondary_commands;
2025-02-27T18:27:58.78181156+07:00 [INFO] perform;
2025-02-27T18:27:58.781855727+07:00 [INFO] authenticate;
2025-02-27T18:27:58.781901042+07:00 [INFO] read_command;
2025-02-27T18:27:58.781922666+07:00 [INFO] perform;
2025-02-27T18:27:58.781938568+07:00 [INFO] authenticate;
2025-02-27T18:27:58.781955924+07:00 [INFO] read_command;
2025-02-27T18:27:58.781990469+07:00 [INFO] perform;
2025-02-27T18:27:58.782013533+07:00 [INFO] authenticate;
2025-02-27T18:27:58.782033544+07:00 [INFO] send_secondary_commands;
2025-02-27T18:27:58.782055502+07:00 [INFO] read_command;
2025-02-27T18:27:58.782130668+07:00 [INFO] read_command;
2025-02-27T18:27:58.782299228+07:00 [INFO] send_secondary_commands;
2025-02-27T18:27:58.782378971+07:00 [INFO] send_secondary_commands;
2025-02-27T18:27:58.782410255+07:00 [INFO] send_secondary_commands;
2025-02-27T18:27:58.782460786+07:00 [INFO] socket reader;
2025-02-27T18:27:58.782487773+07:00 [INFO] read_socket;
2025-02-27T18:27:58.782525957+07:00 [INFO] socket reader;
2025-02-27T18:27:58.782545586+07:00 [INFO] read_socket;
2025-02-27T18:27:58.782567155+07:00 [INFO] read_socket;
2025-02-27T18:27:58.782594647+07:00 [INFO] read_socket;
2025-02-27T18:27:58.782836966+07:00 [INFO] socket reader;
2025-02-27T18:27:58.782859355+07:00 [INFO] read_socket;
2025-02-27T18:27:58.782900418+07:00 [INFO] read_socket;
2025-02-27T18:27:58.78304394+07:00 [INFO] socket reader;
2025-02-27T18:27:58.783066471+07:00 [INFO] read_socket;
2025-02-27T18:27:58.783109576+07:00 [INFO] read_socket;
2025-02-27T18:27:58.783259298+07:00 [INFO] socket reader;
2025-02-27T18:27:58.78327243+07:00 [INFO] read_socket;
2025-02-27T18:27:58.783294988+07:00 [INFO] read_socket;
2025-02-27T18:27:58.78556196+07:00 [INFO] read_socket;
2025-02-27T18:27:58.786437396+07:00 [INFO] Initializing default prettier with plugins {"prettier-plugin-svelte"}
2025-02-27T18:27:58.786471412+07:00 [INFO] Initializing default prettier with plugins {"prettier-plugin-svelte"}
2025-02-27T18:27:58.78774831+07:00 [INFO] read_socket;
2025-02-27T18:27:58.790017282+07:00 [INFO] read_socket;
2025-02-27T18:27:58.791947342+07:00 [INFO] read_socket;
2025-02-27T18:27:58.793805735+07:00 [INFO] read_socket;
2025-02-27T18:27:58.793876421+07:00 [INFO] read_socket;
2025-02-27T18:27:58.793903497+07:00 [INFO] read_socket;
2025-02-27T18:27:58.793932119+07:00 [INFO] read_socket;
2025-02-27T18:27:58.793957983+07:00 [INFO] read_socket;
2025-02-27T18:27:58.794217311+07:00 [INFO] read_socket;
2025-02-27T18:27:58.794235101+07:00 [INFO] read_socket;
2025-02-27T18:27:58.794337077+07:00 [INFO] read_socket;
2025-02-27T18:27:58.794360168+07:00 [INFO] read_socket;
2025-02-27T18:27:58.794515462+07:00 [INFO] read_socket;
2025-02-27T18:27:58.794536313+07:00 [INFO] read_socket;
2025-02-27T18:27:58.794593218+07:00 [INFO] read_socket;
2025-02-27T18:27:58.794952155+07:00 [INFO] read_socket;
2025-02-27T18:27:58.795281291+07:00 [INFO] read_socket;
2025-02-27T18:27:58.795531865+07:00 [INFO] read_socket;
2025-02-27T18:27:58.795818472+07:00 [INFO] read_socket;
2025-02-27T18:27:58.813679907+07:00 [INFO] Initialized prettier with plugins: {"prettier-plugin-svelte"}
2025-02-27T18:27:58.821907737+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:58.827485616+07:00 [INFO] Language server with id 3 sent unhandled notification eslint/status:
{
"uri": "file://~/code/username/optilab/svelte-qwik/sveltekit-app/src/routes/+page.svelte",
"state": 3
}
2025-02-27T18:27:58.893923093+07:00 [INFO] Node runtime install_if_needed
2025-02-27T18:27:58.963055763+07:00 [INFO] starting language server process. binary path: "~/.local/share/zed/node/node-v22.5.1-linux-x64/bin/node", working directory: "~/code/username/optilab/svelte-qwik/sveltekit-app", args: ["~/.local/share/zed/extensions/work/svelte/node_modules/svelte-language-server/bin/server.js", "--stdio"]
2025-02-27T18:27:59.088286244+07:00 [INFO] Language server with id 2 sent unhandled notification @/tailwindCSS/projectInitialized:
[
null
]
2025-02-27T18:27:59.107276088+07:00 [INFO] Language server with id 2 sent unhandled notification @/tailwindCSS/clearColors:
[
null
]

  ```

  </details>


### Zed Version and System Specs

Zed: v0.175.5 (Zed) 
OS: Linux Wayland ubuntu 24.04
Memory: 15.3 GiB
Architecture: x86_64
GPU: Intel(R) Graphics (RPL-P) || Intel open-source Mesa driver || Mesa 24.2.8-1ubuntu1~24.04.1

--- a/crates/copilot/src/copilot.rs
+++ b/crates/copilot/src/copilot.rs
@@ -475,6 +475,7 @@ impl Copilot {
                 binary,
                 root_path,
                 None,
+                Default::default(),
                 cx.clone(),
             )?;
 

--- a/crates/languages/src/go.rs
+++ b/crates/languages/src/go.rs
@@ -5,7 +5,7 @@ use futures::StreamExt;
 use gpui::{App, AsyncApp, Task};
 use http_client::github::latest_github_release;
 pub use language::*;
-use lsp::{InitializeParams, LanguageServerBinary, LanguageServerName};
+use lsp::{LanguageServerBinary, LanguageServerName};
 use project::Fs;
 use regex::Regex;
 use serde_json::json;
@@ -372,14 +372,6 @@ impl super::LspAdapter for GoLspAdapter {
             text: text[display_range].to_string(),
             filter_range,
         })
-    }
-    fn prepare_initialize_params(
-        &self,
-        mut original: InitializeParams,
-    ) -> Result<InitializeParams> {
-        #[allow(deprecated)]
-        let _ = original.root_uri.take();
-        Ok(original)
     }
 }
 

--- a/crates/lsp/src/lsp.rs
+++ b/crates/lsp/src/lsp.rs
@@ -1409,7 +1409,7 @@ impl FakeLanguageServer {
         let server_name = LanguageServerName(name.clone().into());
         let process_name = Arc::from(name.as_str());
         let root = Self::root_path();
-        let workspace_folders = Default::default();
+        let workspace_folders: Arc<Mutex<BTreeSet<Url>>> = Default::default();
         let mut server = LanguageServer::new_internal(
             server_id,
             server_name.clone(),

--- a/crates/lsp/src/lsp.rs
+++ b/crates/lsp/src/lsp.rs
@@ -329,6 +329,7 @@ impl lsp_types::notification::Notification for ServerStatus {
 
 impl LanguageServer {
     /// Starts a language server process.
+    #[allow(clippy::too_many_arguments)]
     pub fn new(
         stderr_capture: Arc<Mutex<Option<String>>>,
         server_id: LanguageServerId,
@@ -1285,7 +1286,6 @@ impl LanguageServer {
 
         let removed: Vec<_> = old_workspace_folders
             .difference(&folders)
-            .into_iter()
             .map(|uri| WorkspaceFolder {
                 uri: uri.clone(),
                 name: String::default(),

--- a/crates/lsp/src/lsp.rs
+++ b/crates/lsp/src/lsp.rs
@@ -336,6 +336,7 @@ impl LanguageServer {
         binary: LanguageServerBinary,
         root_path: &Path,
         code_action_kinds: Option<Vec<CodeActionKind>>,
+        workspace_folders: Arc<Mutex<BTreeSet<Url>>>,
         cx: AsyncApp,
     ) -> Result<Self> {
         let working_dir = if root_path.is_dir() {
@@ -383,6 +384,7 @@ impl LanguageServer {
             code_action_kinds,
             binary,
             root_uri,
+            workspace_folders,
             cx,
             move |notification| {
                 log::info!(
@@ -409,6 +411,7 @@ impl LanguageServer {
         code_action_kinds: Option<Vec<CodeActionKind>>,
         binary: LanguageServerBinary,
         root_uri: Url,
+        workspace_folders: Arc<Mutex<BTreeSet<Url>>>,
         cx: AsyncApp,
         on_unhandled_notification: F,
     ) -> Self
@@ -491,7 +494,7 @@ impl LanguageServer {
             io_tasks: Mutex::new(Some((input_task, output_task))),
             output_done_rx: Mutex::new(Some(output_done_rx)),
             server: Arc::new(Mutex::new(server)),
-            workspace_folders: Default::default(),
+            workspace_folders,
             root_uri,
         }
     }
@@ -617,6 +620,16 @@ impl LanguageServer {
     }
 
     pub fn default_initialize_params(&self, cx: &App) -> InitializeParams {
+        let workspace_folders = self
+            .workspace_folders
+            .lock()
+            .iter()
+            .cloned()
+            .map(|uri| WorkspaceFolder {
+                name: Default::default(),
+                uri,
+            })
+            .collect::<Vec<_>>();
         #[allow(deprecated)]
         InitializeParams {
             process_id: None,
@@ -791,7 +804,7 @@ impl LanguageServer {
                 }),
             },
             trace: None,
-            workspace_folders: Some(vec![]),
+            workspace_folders: Some(workspace_folders),
             client_info: release_channel::ReleaseChannel::try_global(cx).map(|release_channel| {
                 ClientInfo {
                     name: release_channel.display_name().to_string(),
@@ -1260,15 +1273,18 @@ impl LanguageServer {
     }
     pub fn set_workspace_folders(&self, folders: BTreeSet<Url>) {
         let mut workspace_folders = self.workspace_folders.lock();
+
+        let old_workspace_folders = std::mem::take(&mut *workspace_folders);
         let added: Vec<_> = folders
-            .iter()
+            .difference(&old_workspace_folders)
             .map(|uri| WorkspaceFolder {
                 uri: uri.clone(),
                 name: String::default(),
             })
             .collect();
 
-        let removed: Vec<_> = std::mem::replace(&mut *workspace_folders, folders)
+        let removed: Vec<_> = old_workspace_folders
+            .difference(&folders)
             .into_iter()
             .map(|uri| WorkspaceFolder {
                 uri: uri.clone(),
@@ -1276,8 +1292,9 @@ impl LanguageServer {
             })
             .collect();
         let should_notify = !added.is_empty() || !removed.is_empty();
-
         if should_notify {
+            *workspace_folders = folders;
+            drop(workspace_folders);
             let params = DidChangeWorkspaceFoldersParams {
                 event: WorkspaceFoldersChangeEvent { added, removed },
             };

--- a/crates/lsp/src/lsp.rs
+++ b/crates/lsp/src/lsp.rs
@@ -1409,6 +1409,7 @@ impl FakeLanguageServer {
         let server_name = LanguageServerName(name.clone().into());
         let process_name = Arc::from(name.as_str());
         let root = Self::root_path();
+        let workspace_folders = Default::default();
         let mut server = LanguageServer::new_internal(
             server_id,
             server_name.clone(),
@@ -1420,6 +1421,7 @@ impl FakeLanguageServer {
             None,
             binary.clone(),
             root,
+            workspace_folders.clone(),
             cx.clone(),
             |_| {},
         );
@@ -1438,6 +1440,7 @@ impl FakeLanguageServer {
                     None,
                     binary,
                     Self::root_path(),
+                    workspace_folders,
                     cx.clone(),
                     move |msg| {
                         notifications_tx

--- a/crates/prettier/src/prettier.rs
+++ b/crates/prettier/src/prettier.rs
@@ -279,6 +279,7 @@ impl Prettier {
             server_binary,
             &prettier_dir,
             None,
+            Default::default(),
             cx.clone(),
         )
         .context("prettier server creation")?;

--- a/crates/project/src/lsp_store.rs
+++ b/crates/project/src/lsp_store.rs
@@ -206,14 +206,14 @@ impl LocalLspStore {
         );
 
         let binary = self.get_language_server_binary(adapter.clone(), delegate.clone(), true, cx);
-
+        let pending_workspace_folders: Arc<Mutex<BTreeSet<Url>>> = Default::default();
         let pending_server = cx.spawn({
             let adapter = adapter.clone();
             let server_name = adapter.name.clone();
             let stderr_capture = stderr_capture.clone();
             #[cfg(any(test, feature = "test-support"))]
             let lsp_store = self.weak.clone();
-
+            let pending_workspace_folders = pending_workspace_folders.clone();
             move |cx| async move {
                 let binary = binary.await?;
                 #[cfg(any(test, feature = "test-support"))]
@@ -239,12 +239,12 @@ impl LocalLspStore {
                     binary,
                     &root_path,
                     adapter.code_action_kinds(),
+                    pending_workspace_folders,
                     cx,
                 )
             }
         });
 
-        let pending_workspace_folders: Arc<Mutex<BTreeSet<Url>>> = Default::default();
         let startup = {
             let server_name = adapter.name.0.clone();
             let delegate = delegate as Arc<dyn LspAdapterDelegate>;
@@ -7551,10 +7551,11 @@ impl LspStore {
 
         // Update language_servers collection with Running variant of LanguageServerState
         // indicating that the server is up and running and ready
+        let workspace_folders = workspace_folders.lock().clone();
         local.language_servers.insert(
             server_id,
             LanguageServerState::running(
-                workspace_folders.lock().clone(),
+                workspace_folders,
                 adapter.clone(),
                 language_server.clone(),
                 None,

